crates/weaver_checker/src/lib.rs

```diff
@@ -81,6 +113,28 @@ impl WeaverError<Error> for Error {
     }
 }
 
+/// A list of supported policy stages.
+pub enum PolicyStage {
+    /// Policies that are evaluated before resolution.
+    BeforeResolution,
+    /// Policies that are evaluated after resolution.
+    AfterResolution,
+}
+
+impl Display for PolicyStage {
+    /// Returns the name of the policy stage.
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        match self {
+            PolicyStage::BeforeResolution => {
```

Comment:
Nit: can you document PolicyStage and its usage in the README for weaver_checker?

Reply:
I will

Reply:
Done [d2343a3](https://github.com/open-telemetry/weaver/pull/153/commits/d2343a32635148eaab9c4a93e0c6a8c76eed4073)